packages/block-library/src/post-comments/edit.js

```diff
@@ -18,19 +28,56 @@ function PostCommentsDisplay( { postId } ) {
 			// TODO: "No Comments" placeholder should be editable.
 			return comments && comments.length
 				? comments.map( ( comment ) => (
-						<p key={ comment.id }>{ comment.content.raw }</p>
+						<p key={ comment.id }>
+							<RawHTML>{ comment.content.raw }</RawHTML>
+						</p>
```

Comment:
Wrapping `RawHTML` inside a `p` is semantically incorrect because `RawHTML` renders as a `div`.

I think this could be improved in several ways:

1. Use the comment content's `rendered` property instead of `raw`.
`comment.content.raw` is what the user submitted (or, more precisely, what has been saved in the DB after sanitization), and it's supposed to be used when editing the comment.
`comment.content.rendered` is what WP uses when displaying the comment.
In this case, we are not editing but displaying, so `rendered` is more appropriate.
It also automatically wraps paragraphs (as in: blocks of text separated by 2 newlines) in `p` elements! ✨ 

2. Pass a CSS class to the mapped comment (something like `.wp-block-post-comments__comment`), which should make it easier to style it.
Not strictly needed in this PR, but eventually we will need to visually separate comments between each other!

3. Drop the `p` and just return a map of `RawHTML`.


Eventually it would look somewhat like this:
```jsx
return comments.map( comment =>
  <RawHTML className="wp-block-post-comments__comment" key={ comment.id }>
    { comment.content.rendered }
  </RawHTML>
);
```

Reply:
> Wrapping RawHTML inside a p is semantically incorrect because RawHTML renders as a div.

I was under the impression this was not the case:
https://github.com/WordPress/gutenberg/blob/24c6114df4ff3fd18c34fa258a42f1130577f915/packages/element/src/raw-html.js#L23-L24

Reply:
> Use the comment content's rendered property instead of raw.

I do think this makes sense.  I will have to test it out.  I wonder why it was using `raw` in the first place.

Reply:
> I was under the impression this was not the case:

Yeah, I kinda think that comment is not exactly correct. 😄 
Either way, we would be passing more props (the `key`, possibly a `className`), so the `div` should better show up!
_(it does in my tests)_

Reply:
Awesome, these suggestions make things work much better. Thanks!  
The rendered content also has `<br/>`s in it as well which makes it look much better and more in-line with what is on the front end.